appium/tests/specs/mock/inbox/CheckEmptyTrash.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { MockApi } from 'api-mocks/mock';
+import { MockApiConfig } from 'api-mocks/mock-config';
+import {
+  SplashScreen,
+  SetupKeyScreen,
+  MailFolderScreen,
+  EmailScreen,
+  MenuBarScreen
+} from '../../../screenobjects/all-screens';
+
+describe('INBOX: ', () => {
+
+  it('user is able to empty trash and delete all emails', async () => {
+    const mockApi = new MockApi();
+    const testMessageSubject = 'Signed and encrypted message';
+
+    mockApi.fesConfig = MockApiConfig.defaultEnterpriseFesConfiguration;
+    mockApi.ekmConfig = MockApiConfig.defaultEnterpriseEkmConfiguration;
+    mockApi.addGoogleAccount('e2e.enterprise.test@flowcrypt.com', {
+      messages: [testMessageSubject, 'Signed only message'],
+    });
+
+    await mockApi.withMockedApis(async () => {
+      await SplashScreen.mockLogin();
+      await SetupKeyScreen.setPassPhrase();
+      await MailFolderScreen.checkInboxScreen();
+
+      // First find email and delete it
+      await MailFolderScreen.clickOnEmailBySubject(testMessageSubject);
+      await EmailScreen.clickDeleteButton();
+      // Check if deleted email is not displayed in inbox screen
+      await MailFolderScreen.checkEmailIsNotDisplayed(testMessageSubject);
+
+      //  Go to trash screen
+      await MenuBarScreen.clickMenuBtn();
+      await MenuBarScreen.clickTrashButton();
+
+      // First check if mail is displayed in trash screen
+      await MailFolderScreen.checkEmailIsNotDisplayed(testMessageSubject, false);
+      // Empty trash (it would throw error if empty folder button is not present)
+      await MailFolderScreen.emptyFolder();
```

Comment:
@sosnovsky Please check this too.
When user wants to empty trash (delete all messages), mock api returns below error.

> [0-14] 1ms | 400 GET /gmail/v1/users/me/messages?labelIds=TRASH&maxResults=500&prettyPrint=false | {"mockError":"Method not implemented for /gmail/v1/users/me/messages?labelIds=TRASH&maxResults=500&prettyPrint=false: GET","mockStack":"no stack","noFormatter":"if you wanted a different error format, subclass from HttpErr and add a formatter"}

Reply:
Fixed it ✅

Reply:
Thanks